http/src/components/FriendsList.js

```diff
@@ -1,11 +1,10 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { getFriends } from '../actions';
-import axios from 'axios';
 
 class FriendsList extends Component {
-    componentDidMount() {
-        this.props.getFriends();
+    componentDidMount(props) {
```

Comment:
There's no need to pass props in this case, to either `componentDidMount()` or `this.props.getFriends()`. I would actually remove them. It might be confusing for other people who read your code.